quarkus/config-api/src/main/java/org/keycloak/config/LoggingOptions.java

```diff
@@ -158,40 +161,58 @@ public String toString() {
 
     public static final Option<String> LOG_SYSLOG_ENDPOINT = new OptionBuilder<>("log-syslog-endpoint", String.class)
             .category(OptionCategory.LOGGING)
-            .description("The IP address and port of the syslog server.")
+            .description("Set the IP address and port of the Syslog server.")
             .defaultValue("localhost:514")
             .build();
 
+    public static final Option<String> LOG_SYSLOG_TYPE = new OptionBuilder<>("log-syslog-type", String.class)
+            .category(OptionCategory.LOGGING)
+            .expectedValues(Arrays.stream(SyslogHandler.SyslogType.values()).map(f -> f.toString().toLowerCase()).toList())
+            .description("Set the Syslog type used to format the sent message.")
+            .defaultValue(SyslogHandler.SyslogType.RFC5424.toString().toLowerCase())
+            .build();
+
+    public static final Option<Level> LOG_SYSLOG_LEVEL = new OptionBuilder<>("log-syslog-level", Level.class)
+            .category(OptionCategory.LOGGING)
+            .defaultValue(Level.ALL)
+            .description("Set the log level specifying which message levels will be logged by the Syslog logger. Message levels lower than this value will be discarded.")
```

Comment:
Should we perhaps mention, if/how this plays with `log-level`? Which of these take precedence?